helm/ffc-sfd-business/values.yaml

```diff
@@ -28,7 +28,7 @@ livenessProbe:
 # These may be changed. See here: https://github.com/DEFRA/adp-helm-library?tab=readme-ov-file#container-template
 readinessProbe:
   path: /healthy
```

Comment:
does this need to be prefixed with `/business`

Reply:
Yep